package-lock.json

```diff
@@ -104,6 +104,33 @@
         "js-tokens": "3.0.2"
```

Comment:
What do you think about removing the `package-lock.json`? Having it could create confusion for devs as we have the `yarn.lock` as our primary lock file. I think a `yarn` would not update the `package-lock.json`, am I right?

Reply:
Oops, that was a mistake.